site/it/tutorials/quickstart/advanced.ipynb

```diff
@@ -0,0 +1,418 @@
+        "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+        "\n",
+        "# Add a channels dimension\n",
+        "x_train = x_train[..., tf.newaxis]\n",
+        "x_test = x_test[..., tf.newaxis]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "k1Evqx0S22r_"
+      },
+      "source": [
+        "Usa `tf.data` per raggruppare e mischiare il dataset:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "8Iu_quO024c2",
+        "colab": {}
+      },
+      "source": [
+        "train_ds = tf.data.Dataset.from_tensor_slices(\n",
+        "    (x_train, y_train)).shuffle(10000).batch(32)\n",
+        "\n",
+        "test_ds = tf.data.Dataset.from_tensor_slices((x_test, y_test)).batch(32)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BPZ68wASog_I"
+      },
+      "source": [
+        "Costrusci il modello `tf.keras` usando l'[API Keras per creare sottoclassi di modelli](https://www.tensorflow.org/guide/keras#model_subclassing):"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "h3IKyzTCDNGo",
+        "colab": {}
+      },
+      "source": [
+        "class MyModel(Model):\n",
+        "  def __init__(self):\n",
+        "    super(MyModel, self).__init__()\n",
+        "    self.conv1 = Conv2D(32, 3, activation='relu')\n",
+        "    self.flatten = Flatten()\n",
+        "    self.d1 = Dense(128, activation='relu')\n",
+        "    self.d2 = Dense(10, activation='softmax')\n",
+        "\n",
+        "  def call(self, x):\n",
+        "    x = self.conv1(x)\n",
+        "    x = self.flatten(x)\n",
+        "    x = self.d1(x)\n",
+        "    return self.d2(x)\n",
+        "\n",
+        "# Create an instance of the model\n",
+        "model = MyModel()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "uGih-c2LgbJu"
+      },
+      "source": [
+        "Scegli un metodo di ottimizzazione e una funzione obiettivo per l'addestramento:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "u48C9WQ774n4",
+        "colab": {}
+      },
+      "source": [
+        "loss_object = tf.keras.losses.SparseCategoricalCrossentropy()\n",
+        "\n",
+        "optimizer = tf.keras.optimizers.Adam()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JB6A1vcigsIe"
+      },
+      "source": [
```

Comment:
Precisione or accuratezza? It Is Better to use the same. We have something similar in the same document